### PR TITLE
Add Tap Forms Mac 5.app

### PR DESCRIPTION
**Pull request by johnpitchko**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/tap-forms.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'tap-forms' do
+  version '5.3.11'
+  sha256 '4dd6383a90108a754914819ee0452ac53c05ead7b004a38e8bf243ca956bb0a7'
+
+  # paddle.com was verified as official when first introduced to the cask. Developer uses paddle.com to process licences and payments for the software
```
Comment (miccal) [inline Casks/tap-forms.rb:5]: Please change this to
```
vendors.paddle.com/download/product/503174 was verified as official when first introduced to the cask
```

File: Casks/tap-forms.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  url 'https://vendors.paddle.com/download/product/503174'
```
Comment (miccal) [inline Casks/tap-forms.rb:6]: Please add the following [`appcast`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md):
```
appcast "https://www.tapforms.com/app/tf-mac-#{version.major_minor}-sparkle-appcast.xml"
```

File: Casks/tap-forms.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Tap Forms 5'
+  homepage 'https://www.tapforms.com/'
+
+  app 'Tap Forms Mac 5.app'
```
Comment (miccal) [inline Casks/tap-forms.rb:10]: Please change this to
```
app "Tap Forms Mac #{version.major}.app"
```

File: Casks/tap-forms.rb (hunk 2 above)
Comment (johnpitchko) [reply Casks/tap-forms.rb:6]: Updated; I also learned about the script to find appcasts, so I'll be sure to run that when adding future casks.

File: Casks/tap-forms.rb (hunk 1 above)
Comment (johnpitchko) [reply Casks/tap-forms.rb]: Updated; sorry didn't know I needed the full URL.

File: Casks/tap-forms.rb (hunk 3 above)
Comment (johnpitchko) [reply Casks/tap-forms.rb]: Updated

**Review by johnpitchko — COMMENTED** [review]
Updated based on @miccal 's comments. Thank you!

**Comment by ran-dall** [comment]
@johnpitchko Thanks for the submission however this is behind a paywall (Paddle), so it needs to be submitted the the `unofficial` tap.

> The download URL for the app is both behind a login/registration form and from a host that differs from the homepage, meaning users can’t easily verify its authenticity. alehouse/homebrew-unofficial is a sister repo where you may wish to submit your cask.

**Comment by johnpitchko** [comment]
Hi @ran-dall, I checked the URL and it is not behind a paywall. The URL is the direct download to the DMG to install the app. Yes, the app is trialware, but it is fully functioning for the 15 day trial period. No login/registration is required to download the app. Perhaps it needs to go in a different tap (placing in a different tap wasn't super clear to me in the instructions so perhaps I got that wrong)??

**Comment by ran-dall** [comment]
@johnpitchko Thanks for the clarification. Normally, Paddle is normally used to download something after you purchase it. I wasn't aware you can openly distribute software from it.

**Comment by johnpitchko** [comment]
Thanks a bunch @ran-dall! I'll see if I can figure out why CI is failing.

**Comment by ran-dall** [comment]
@johnpitchko It's failing right now because of some updates that were done. Don't worry about the `CI` right now. If you want you can run it locally with this command 
```shell
$ /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/developer/bin/cask-pr-local-check https://github.com/Homebrew/homebrew-cask/pull/84017
```

**Comment by ran-dall** [comment]
```shell
$ /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/developer/bin/cask-pr-local-check https://github.com/Homebrew/homebrew-cask/pull/84017
==> Downloading https://vendors.paddle.com/download/product/503174
Already downloaded: /Users/randall/Library/Caches/Homebrew/downloads/9943b264d1cf786f64a3b20e0b8920049bd96d3373eef0c18b76f0b73260e6eb--oWMXZJBQwqBK97wQ1Lw3_Tap Forms Install 5.3.11.dmg
==> Verifying SHA-256 checksum for Cask 'tap-forms'.
audit for tap-forms: passed

1 file inspected, no offenses detected
```

**Pull request merged**